src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -120,14 +119,14 @@ public virtual void WriteMessage(LogLevel logLevel, string logName, int eventId,
                 logLevelColors = GetLogLevelConsoleColors(logLevel);
                 logLevelString = GetLogLevelString(logLevel);
                 // category and event id
-                logIdentifier = _loglevelPadding + logName + "[" + eventId + "]";
+                sb.AppendLine(_loglevelPadding + logName + "[" + eventId + "]");
```

Comment:
Would this be even faster if you would `sb.Append` all the time instead of string concat?